app/src/main/java/com/github/swent/echo/viewmodels/association/AssociationViewModel.kt

```diff
@@ -119,3 +136,10 @@ constructor(
         viewModelScope.launch { allEvents = repository.getAllEvents() }
     }
 }
+
+// Status of the Association screen/viewmodel
+sealed class AssociationSatus {
+    data object Okay : AssociationSatus()
+
+    data object Error : AssociationSatus()
```

Comment:
I thought about that but since it's the only message handled for now I don't think it's needed. It's relatively straigt-forward to extend this later in case it would be needed.

Reply:
That's a good point